openseadragon/openseadragon.js

```diff
@@ -4062,8 +4062,7 @@ $.EventSource.prototype = {
     }
 
     // Note: window.navigator.pointerEnable is deprecated on IE 11 and not part of W3C spec.
-    // if ( window.PointerEvent && ( window.navigator.pointerEnabled || $.Browser.vendor !== $.BROWSERS.IE ) ) {
-    if (false){
+    if ( window.PointerEvent && ( window.navigator.pointerEnabled || $.Browser.vendor !== $.BROWSERS.IE ) ) {
```

Comment:
no, actually the opposite. That modification reverses OpenSeadragon to vanilla and modifies fabric instead